runtime/near-vm-runner/src/tests/compile_errors.rs

```diff
@@ -138,6 +139,7 @@ fn test_evil_function_index() {
 }
 
 #[test]
+#[allow(deprecated)]
```

Comment:
Move all the `#[allow(deprecated)]` right before `ProtocolFeature::_DeprecatedPreparationV2` across all the three test files.